en/services/opendroneid.md

```diff
@@ -134,8 +134,9 @@ The MAVLink components in the UAS involved in the drone ID MAVLink message excha
 If the `HEARTBEAT` messages are not received within a required time interval, they must declare a malfunction of the Remote ID system and indicate this in the `status` field of the [LOCATION](#OPEN_DRONE_ID_LOCATION) message.
 This must also be indicated to the operator of the UAS.
 
-Similarly, the autopilot and GCS components must listen to the [ARM_STATUS](#OPEN_DRONE_ID_ARM_STATUS) from the RID transmitter component(s) and not allow the UA to be airborne before the RID transmitter component(s) is ready.
-During flight, if the arm status indicates a failure, similar action must be taken as for a lack of `HEARTBEAT` messages.
+Similarly, the autopilot must listen to the [ARM_STATUS](#OPEN_DRONE_ID_ARM_STATUS) from the RID transmitter component(s) and not allow the UA to be airborne before the RID transmitter component(s) is ready.
+During flight, if the arm status indicates a failure, similar action must be taken as for a lack of `HEARTBEAT` messages from the RemoteID.
+The [ARM_STATUS](#OPEN_DRONE_ID_ARM_STATUS) message must also be routed to a GCS, if present, allowing it to  provide more detailed information about RemoteID arming failures.
```

Comment:
So this now says the autopilot must route to the GCS, if present. That makes sense to me unless Remote ID flight _requires_ a GCS (which would be stupid). 

I'm not even sure if we should be this strong in the wording. YOu might have high latency connection to a balloon - a GCS is technically connected but you could not comply with this requirement. 
I think the routing should be a "should".

Reply:
I think I wrote the original text under the wrong assumption that the GCS is the part that decides whether the UA can take off or not. I agree with your changes to correct that.

Reading the MOC text, I do get the impression that it is mandatory to route this status info the GCS (when it is present) so I think it would be fair to leave the text as it now currently is in the PR.

Reply:
Thanks very much. The way I read that spec you're right - it assumes all flights are monitored. We probably should say "Controller" rather than GCS though, because they allow for an RC handheld with just a light or some other indicated. 
Good enough for now though. 

Much appreciated.